actions/websocket_actions.jsx

```diff
@@ -538,6 +542,20 @@ function handlePostDeleteEvent(msg) {
     dispatch(postDeleted(post));
 }
 
+async function handlePostUnreadEvent(msg) {
+    await dispatch(
```

Comment:
The async/await seems unnecessary here since nothing is waiting on `handlePostUnreadEvent`

Reply:
If you want this for testing, you can also just return the results of the `dispatch` so that the caller can await on the `Promise` if they want. I'm only 0/5 on that vs what you're currently doing though since I know some people prefer this way because it makes it very clear that the function returns an async result